content/sensu-go/5.15/reference/checks.md

```diff
@@ -175,9 +175,16 @@ spec:
 #### Cron scheduling
 
 You can also schedule checks using [cron syntax][14].
-For example, to schedule a check to execute once a minute at the start of the minute, set the `cron` attribute to `* * * * *` and the `publish` attribute to `true`.
 
-**Example cron check**
+Examples of valid cron values include:
+
+- `cron: CRON_TZ=Asia/Tokyo 0 * * * *`
+- `cron: TZ=Asia/Tokyo 0 * * * *`
+- `cron: 0 * * * *`
+
+**Example cron checks**
+
+To schedule a check to execute once a minute at the start of the minute, set the `cron` attribute to `0 * * * *` and the `publish` attribute to `true`.
```

Comment:
Once a minute (on the minute): `* * * * *`
Once an hour (on the hour): `0 * * * *`
See https://en.wikipedia.org/wiki/Cron

```suggestion
To schedule a check to execute once a minute at the start of the minute, set the `cron` attribute to `* * * * *` and the `publish` attribute to `true`.
```